ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -56,9 +56,7 @@
   {:collection-draft :umm-c
    :service-draft :umm-s
    :tool-draft :umm-t
-   :variable-draft :umm-var
-   :data-quality-summary-draft :data-quality-summary
```

Comment:
I'm assuming that the mapping for the generic concept type to draft concept type can just be pulled out and is being done so somewhere else